query-engine/core/src/executor/mod.rs

```diff
@@ -116,20 +115,3 @@ pub trait TransactionManager {
     /// Rolls back a transaction.
     async fn rollback_tx(&self, tx_id: TxId) -> crate::Result<()>;
 }
-
-// With the node-api when a future is spawned in a new thread `tokio:spawn` it will not
-// use the current dispatcher and its logs will not be captured anymore. We can use this
-// method to get the current dispatcher and combine it with `with_subscriber`
-// let dispatcher = get_current_dispatcher();
-// tokio::spawn(async {
-//      my_async_ops.await
-// }.with_subscriber(dispatcher));
-//
-//
-// Finally, this can be replaced with with_current_collector
-// https://github.com/tokio-rs/tracing/blob/master/tracing-futures/src/lib.rs#L234
-// once this is in a release
-
-pub fn get_current_dispatcher() -> Dispatch {
-    tracing::dispatcher::get_default(|current| current.clone())
-}
```

Comment:
Opportunistic refactoring. `with_current_subscriber` actually does exist even now in stable tracing 0.1. Maybe it didn't back when it was written, and it was backported from the main branch later?